### ISSUE TITLE
Publish scalafix-testkit for Scala 2.12.9.

### ISSUE THREAD

**Issue by kamilkloch** [scalacenter/scalafix#978]
Currently, only 2.12.8 is supported: https://mvnrepository.com/artifact/ch.epfl.scala/scalafix-testkit.

**Comment by gabro** [comment]
I suspect this is blocked by https://github.com/scalameta/scalameta/pull/1893

**Comment by aeons** [comment]
Scalameta is out for 2.12.9 now.

**Comment by aeons** [comment]
Any update on this? The latest release notes mention support for 2.12.9, so does this just need a publish of the current code?

**Comment by olafurpg** [comment]
The 2.12.9 upgrade is done in https://github.com/scalacenter/scalafix/pull/980 alongside 2.13. A workaround in the meantime could be:

```scala
libraryDependencies += "ch.epfl.scala" % "scalafix-testkit_2.12.8" % "..."
```

**Comment by aeons** [comment]
That works for testkit, but testkit depends on scalafix-cli, which it still tries to resolve as `ch.epfl.scala:scalafix-cli_2.12.9:0.9.6`.
I guess we need to run our scalafix project under 2.12.8.

**Comment by aeons** [comment]
Yep, it's there now. Thanks. Took several hours to sync from sonatype though.
